node/setup.go

```diff
@@ -700,7 +700,7 @@ func createPEXReactorV2(
 	logger log.Logger,
 	peerManager *p2p.PeerManager,
 	router *p2p.Router,
-) (*pex.ReactorV2, error) {
+) (service.Service, error) {
```

Comment:
Mind explaining why we should change this return? It seems like we can just as easily use the result as a `service.Service` without losing the extra type information that having the return value be typed would give us?

Reply:
this seemed easier, I think this is just an intermediate step, as (eventually) the constructor will move into the pex package and it'll just be a service at that point. 

We don't need (or want) the extra type information.